src/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 400;
+  font-style: normal;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin: 20px auto 0;
+}
+
+.search-bar_big {
+  height: 70px;
+  width: 100%;
+}
+
+.search-bar_small {
+  height: 42px;
+  width: 100%;
+}
+
+.search-bar__input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
```

Comment:
We could extract all of the colors to variables to make them reusable

Reply:
done! :)